vtex-sites/base.store
# fix: allows for GTM debugging

## What's the purpose of this pull request?
This PR makes GTM debugging work properly.

## How it works? 
Since the debugging script GTM uses breaks inside partytown, we're using the `gtm_debug` query string as a flag for enabling/disabling partytown.

## How to test it?
Include the gtm_debug query string on any page of the website and check if the GTM script loads outside partytown (no text/partytown type on the script tag).

## References
We've tried to make it work using partytown directly, but with no success. The partytown team seems to be fixing the issues.
